zephyr/module.yml

```
@@ -197,9 +197,9 @@ blobs:
     description: "Modified bootrom to boot Blackhole"
     doc-url: https://tenstorrent.com
   - path: fw_pack-wormhole.tar.gz
-    sha256: 824d86a2ff0dd195e9a71bc85211e1b06b88332e1859142f9eeef1f828642001
+    sha256: 14f481be865e944327fc9fdc77a30a7fb1abb45ab7783923d675d9bbc7562a59
     type: img
-    version: '18.12.0-rc1'
+    version: '19.0.0-rc3'
```

Hopefully not overly optimistic, but I think our next version will be 19.0.0, so let's use that here
```suggestion
    version: '19.0.0'
```